docs/topics/api/accounts.rst

```diff
@@ -2,64 +2,77 @@
+.. http:get:: /api/v3/accounts/account/(int:user_id)/
 
-    .. sourcecode:: bash
+    .. _account-object:
 
-        curl "https://addons.mozilla.org/api/v3/accounts/profile/"
-            -H "Authorization: JWT <jwt-token>"
+    :>json int id: The numeric user id.
+    :>json string username: username chosen by the user, used in the account url. If not set will be a randomly generated string.
+    :>json string name: The name chosen by the user, or the username if not set.
+    :>json float average_addon_rating: The average rating for addons the developer has listed on the website.
+    :>json int num_addons_listed: The number of addons the developer has listed on the website.
+    :>json string|null biography: More details about the user.
+    :>json string|null homepage: The user's website.
+    :>json string|null location: The location of the user.
+    :>json string|null occupation: The occupation of the user.
+    :>json string picture_url: URL to a photo of the user, or `/static/img/anon_user.png` if not set.
+    :>json string|null picture_type: the image type (only 'image/png' is supported) if a user defined photo has been provided, or none if no photo has been provided.
+    :>json boolean is_addon_developer: The user has developed and listed add-ons on this website.
+    :>json boolean is_artist: The user has developed and listed themes on this website.
 
-    **Response:**
 
-    .. sourcecode:: json
-
-        {
-            "username": "bob",
-            "display_name": "bob",
-            "email": "a@m.o",
-            "biography": "Some biography",
-            "deleted": false,
-            "display_collections": false,
-            "display_collections_fav": false,
-            "homepage": "https://a.m.o",
-            "location": "Vancouver",
-            "notes": null,
-            "occupation": "",
-            "picture_type": "",
-            "picture_url": "/static/img/anon_user.png",
-            "read_dev_agreement": "2015-11-20T18:36:12",
-            "is_verified": true
-        }
-
-    :statuscode 200: profile found.
+    :statuscode 200: account found.
     :statuscode 400: an error occurred, check the `error` value in the JSON.
-    :statuscode 401: authentication failed.
+    :statuscode 404: no account with that user id.
 
 
 .. important::
 
-    * Biography can contain HTML, or other unsanitized content and it's the
-      responsibiliy of the client to clean it appropriately before display.
+    * `Biography` can contain HTML, or other unsanitized content, and it is the
+      responsibiliy of the client to clean and escape it appropriately before display.
+
+
+------------
+Self Account
+------------
+
+.. _`self-account`:
+
+If you authenticate and access your own account (either by specifing your own user_id, or omiting it) the following additional fields are returned.
+If you have `Users:Edit` permission you will see these extra fields for all user accounts.
+
+.. http:get:: /api/v3/accounts/account/
```

Comment:
Just having `/accounts/profile/` redirect to `/accounts/account/<userid>/` should be enough - or, if we really want a specific endpoint, make `/accounts/profile/` a simple view that calls AccountViewSet detail method behind the scenes instead of a redirect.